src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -10,7 +10,7 @@ const INITIAL_STATUS: CopyToClipboardStatus = 'pending';
 export function CopyToClipboard(props: CopyToClipboardProps) {
     const {children, text, timeout, onCopy} = props;
 
-    const textRef = React.useRef(text);
+    const textRef = React.useRef('');
```

Comment:
didnt actually understand why do we need text here
looks like it not used anywhere outside of onClickWithCopy